src/modules/person/persistence/person.repository.ts

```diff
@@ -257,7 +257,7 @@ export class PersonRepository {
         if (!keyCloakUserDataResponse.ok) {
             return person;
         }
-        person.userLock = (await this.userLockRepository.findPersonById(person.id)) ?? undefined;
+        person.userLock = (await this.userLockRepository.findByPersonId(person.id)) ?? undefined;
         person.isLocked = keyCloakUserDataResponse.value.enabled === false;
```

Comment:
`findByPersonId` always returns an array, don't default it to `undefined` (Which won't even do anything)